numeric.c

```diff
@@ -4777,12 +4777,20 @@ int_xor(VALUE x, VALUE y)
 }
 
 /*
- * Document-method: Integer#<<
- * call-seq:
- *   int << count  ->  integer
+ *  call-seq:
+ *    int << count -> integer
```

Comment:
Don't we typically use `self` as the receiver in this case, instead of `int`?  Same issue with `>>`.

Reply:
There were three of these, now fixed.